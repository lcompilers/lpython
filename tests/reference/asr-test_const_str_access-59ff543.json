{
    "basename": "asr-test_const_str_access-59ff543",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_const_str_access.py",
    "infile_hash": "356d24d72537679b048256fe36e06defe1525ea6abbdef9d9a05bce7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_const_str_access-59ff543.stderr",
    "stderr_hash": "22aab31426a34d40549295ecdbdac10ecdf67689aa464522f42b0f52",
    "returncode": 2
}