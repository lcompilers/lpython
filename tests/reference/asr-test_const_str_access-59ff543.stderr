semantic error: 'Const' not required as strings are already immutable
 --> tests/errors/test_const_str_access.py:5:19
  |
5 |     CONST_STRING: Const[str] = "Hello, LPython!"
  |                   ^^^^^^^^^^ 
